### ISSUE TITLE
NaN FloatDomainError (Reopen of #15)

### ISSUE THREAD

**Issue by softwaregravy** [colszowka/simplecov#23]
https://github.com/colszowka/simplecov/issues/closed#issue/15

Okay, so I was seeing this for a while.  It was almost reproduce-able, in that it was (I could add a pending to a test, then uncomment the pending and it would pass and fail deterministically.)  So I took my code, and prepared to present it here to help find the problem ... and while doing so ... it started passing every time.  So, the error definitely appears to be non-deterministic.

Anyway, here's the code
https://github.com/softwaregravy/base_project  (first checkin and first change)
And here's the change.  I just kept doing and undoing this and saw the error
https://github.com/softwaregravy/base_project/commit/896d527c20557f99a86fd2d47db0873db39f0327

I was also installing and uninstalling annotate-models 1.0.4 which was the last gem I had installed before this happened (but no, I have no idea how that might impact it, just a reaction like "oh no, what did I change last").  First observed while running autotest.

It does not now appear to be reproduce-able.  Weird.  Anything pulling timestamps in the code?

I was in the process of factoring out my simplecov config (so there's some code there, probably in the wrong place, but it was not in use).

So, in conclusion, I observed the failure, was able to reproduce for a short while, but not cannot.  Not vary satisfying, I know.
